rct229/ruletest_engine/ruletest_jsons/section23/rule_23_7.json

```diff
@@ -0,0 +1,198 @@
+{
+    "rule-23-7-a": {
+        "Section": 23,
+        "Rule": 7,
+        "Test": "a",
+        "test_description": "System 6's fan system correctly specified temperature control as constant",
+        "expected_rule_outcome": "pass",
+        "standard": {
+            "rule_id": "23-7",
+            "ruleset_reference": "G3.1.3.14",
+            "rule_description": "For baseline systems 6 and 8,  supply air temperature setpoint shall be constant at the design condition",
+            "applicable_rmr": "Baseline RMR",
+            "rule_assertion": "=",
+            "comparison_value": "Expected Value",
+            "rule_dependency": "System 6, 8 (Rule18-a, to be confirmed)",
+            "mandatory_rule": "Yes",
+            "schema_version": "0.0.23"
+        },
+        "rmr_transformations": {
+            "baseline": {
+                "id": "ASHRAE229 1",
+                "ruleset_model_instances": [
+                    {
+                        "id": "RMD 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_open_schedule": "Required Building Schedule 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                                "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                                "terminals": [
+                                                    {
+                                                        "id": "VAV Air Terminal 1",
+                                                        "is_supply_ducted": true,
+                                                        "type": "VARIABLE_AIR_VOLUME",
+                                                        "served_by_heating_ventilating_air_conditioning_system": "System 6",
+                                                        "heating_source": "ELECTRIC"
```

Comment:
Juan, could you also add '''fan_configuration''' here?

Reply:
Thanks Yun Joon, I missed one of the terminals. Updated!